renderdoc/core/core.cpp

```diff
@@ -734,6 +734,7 @@ RDCFile *RenderDoc::CreateRDC(RDCDriver driver, uint32_t frameNum, void *thpixel
     th.pixels = (const byte *)thpixels;
     th.width = thwidth;
     th.height = thheight;
+    th.format = thformat;
```

Comment:
Should we assert here that the thumbnail is a 'regular' type? E.g. if it was `DDS` we don't handle that, similar for `HDR/EXR`. It looks like we probably want to ensure that it's one of `JPG`, `Raw`, `PNG`, `BMP`, `TGA`.

Reply:
Done